### PR TITLE
Bug fix/privileges after signout

### PR DESCRIPTION
**Pull request by Jaakik**
* Bug is Fixed by moving directly to Homefragemnt after SignOut 
* Some Kdoc for signingWidget was added
* Removed Unecessary code to update the currentUser  in UpdateUI as it is not used anywhere else, the initial idea was to preserve consistency

### CODE REVIEW

**Comment by Jaakik** [comment]
@mlenw I didn't deal with the BackPress as it seemed to work fine When testing in on my phone, please request any changes you think are necessary in this regard.

**Review by NicolasVial — APPROVED** [review]
Looks good to me but... are you sure about the KDOC? I mean... I think Jonas have a PR to add all the missing KDoc ^^' we don't want to have conflicts in the KDoc I think :)

File: app/src/main/res/navigation/nav_graph.xml [hunk 1]
```diff
@@ -172,7 +172,11 @@
         android:id="@+id/googleSignInWidget"
         android:name="com.github.HumanLearning2021.HumanLearningApp.view.GoogleSignInWidget"
         android:label="@string/nav_graph_label_login"
-        tools:layout="@layout/fragment_google_sign_in" />
+        tools:layout="@layout/fragment_google_sign_in" >
+        <action
+            android:id="@+id/action_googleSignInWidget_to_homeFragment"
+            app:destination="@id/homeFragment" />
```
Comment (martin-lenweiter) [inline app/src/main/res/navigation/nav_graph.xml]:
```suggestion
            app:destination="@id/homeFragment"
            app:popUpTo="@+id/homeFragment"
            app:popUpToInclusive="true"
            />
```

Comment (martin-lenweiter) [reply app/src/main/res/navigation/nav_graph.xml]: When I press back it takes me back to login fragment. I think this should fix it.

**Review by martin-lenweiter — CHANGES_REQUESTED** [review]
Back thing has an issue, and I agree with Nicolas about Kdoc. Need to be careful with merge conflicts. Other than that all good

**Comment by Jaakik** [comment]
> Looks good to me but... are you sure about the KDOC? I mean... I think Jonas have a PR to add all the missing KDoc ^^' we don't want to have conflicts in the KDoc I think :)

Okay I will check that

**Review by martin-lenweiter — APPROVED** [review]

**Pull request merged**